client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -48,7 +48,7 @@ public class RemoteShuffleMaster implements ShuffleMaster<RemoteShuffleDescripto
   private final ScheduledThreadPoolExecutor executor =
       new ScheduledThreadPoolExecutor(
           1,
-          ThreadUtils.createFactoryWithDefaultExceptionHandler(
+          org.apache.celeborn.common.util.ThreadUtils.namedThreadFactory(
```

Comment:
any conflicts? why full package name is required?

Reply:
@pan3793, `RemoteShuffleMaster` also invokes `org.apache.celeborn.plugin.flink.utils.ThreadUtils#shutdownExecutors`.